.github/ISSUE_TEMPLATE/001_bug_report.md

```diff
@@ -27,31 +29,44 @@ the problem you're facing:
  - Is this a regression? If yes, have you been able to "git bisect" it to a
    specific commit?
  - ...
+-->
 
 **To Reproduce**
+<!--
 Steps to reproduce the behavior:
 1. mkdir build; cd build
 2. cmake -DBOARD=board\_xyz
 3. make
 4. See error
+-->
```

Comment:
Shouldn't this be left as-is for people to fill in?

Reply:
that, I am not sure we should leave, the text above is not fill-in really, there are different ways steps can added, the text serves as example.

Reply:
That's fair, but can we at least leave the SHA, platform, and toolchain visible?

Reply:
yes, I did that already, forgot to post comment...